### PR TITLE
Added Padding in the (Community Discussion Forum) button on the chapter's page (In Mobile devices) 

### LINKED ISSUE #91: No padding in the (Community Discussion Forum) button on the chapter's page (In Mobile devices)
#### Description

No padding in the Community Discussion Forum on the chapter's page (In Mobile devices)

#### Expected Behavior

Before
Before
After
After

#### Environment:
- Meshery Version: 
- Kubernetes Version: 
- Host OS:
- Browser:

### PR DESCRIPTION
**Pull request by UtkarshMishra12**
Signed-off-by: Utkarsh Mishra <nasautkarsh@gmail.com>

**Description**
Added padding in the (Community Discussion Forum) button on the chapter's page (In Mobile devices) 

This PR fixes #91 
image

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ace0a1c2f3db7bc3b92b05f43685ffe3ce4c880d at: https://63700b8c075a5f3300d40b44--service-mesh-patterns.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 721be1d76cb649a34355fc2698cbd11a67ca8cbd at: https://6370e6a12a3714418bfb45b1--service-mesh-patterns.netlify.app

File: site/src/sections/Chapters/chapters.style.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ const ChaptersStyleWrapper = styled.main`
     }
     @media (max-width: 1000px) {
       flex-direction: column; 
+      padding:20px;
```
Comment (Chadha93) [inline site/src/sections/Chapters/chapters.style.js:51]: Please use rem.

**Comment by UtkarshMishra12** [comment]
@Chadha93 I have done the changes, but the build preview site is failing.

**Comment by UtkarshMishra12** [comment]
@Chadha93 Gentle reminder 😄

**Comment by l5io** [comment]
🚀 Preview for commit a547f054070648949235b735dfbefee46eb20890 at: https://63766e256cf2ba48bbc496ef--service-mesh-patterns.netlify.app

**Comment by UtkarshMishra12** [comment]
@Chadha93 Now it's completed. You can check it.

**Review by Chadha93 — APPROVED** [review]
LGTM, Good work @UtkarshMishra12 :1st_place_medal:

**Pull request merged**